src/cascadia/TerminalSettingsEditor/Actions.xaml

```diff
@@ -331,10 +186,14 @@
                     HorizontalAlignment="Left"
                     Spacing="8"
                     Style="{StaticResource SettingsStackStyle}">
+            <HyperlinkButton x:Uid="Actions_Disclaimer"
+                             Margin="0"
+                             Padding="0"
+                             NavigateUri="https://learn.microsoft.com/en-us/windows/terminal/customize-settings/actions" />

```

Comment:
two things. make this a fwlink (not an aka link! you still make them on https://aka.ms, but it's a different category) and drop the en-us (so that the docs site can choose the most appropriate language for the user!)